src/transformers/training_args.py

```diff
@@ -456,6 +456,11 @@ class TrainingArguments:
         torchdynamo (`str`, *optional*):
             The token that is used to set the backend compiler for TorchDynamo. Possible choices are ["eager",
             "nvfuser]. This is an experimental API and subject to change.
+        ray_scope (`str`, *optional*, defaults to `"last"`):
+            The scope to use when doing hyperparameter search with Ray. By default, `"last"` will be used. Ray will
+            then use the last checkpoint of all trials, compare those, and select the best one. However, other options
+            are also available. See the [Ray documentation]( https://docs.ray.io/en/latest/tune/api_docs/analysis.html\
+#ray.tune.ExperimentAnalysis.get_best_trial) for more options.
```

Comment:
This formating is intentional to satisfy line length limits, see https://stackoverflow.com/questions/14414878/how-do-i-break-a-link-in-a-rst-docstring-to-satisfy-pep8 for example

Reply:
Yes but we do not use rst docstrings, so this won't be rendered in our doc. Just put everything back in one line and `make style` will automatically format it the best it can be.

Reply:
cc @BramVanroy